app/javascript/packs/red_hat_cloud_services_redirect.jsx

```diff
@@ -3,7 +3,7 @@ import { Button } from 'patternfly-react'
 
 const RedHatCloudServicesRedirect = () => {
   return (
-    <div>
+    <div id='red_hat_cloud_services_redirect'>
```

Comment:
Can an ID be put in the `<p>` below too? It would make locating that subtext easier